src/openrct2/actions/LargeSceneryPlaceAction.cpp

```diff
@@ -275,6 +275,7 @@ GameActions::Result::Ptr LargeSceneryPlaceAction::Execute() const
             {
                 wall_remove_at({ curTile, zLow, zHigh });
             }
+            MapInvalidateHeightCache();
```

Comment:
Should this check for `if (!(GetFlags() & GAME_COMMAND_FLAG_GHOST))` ?

Reply:
It does; It's wrapped with an if block checking that 7 lines above.